app/serializers/search_and_compare/course_serializer.rb

```diff
@@ -71,7 +71,7 @@ class CourseSerializer < ActiveModel::Serializer
     # Campuses_related_Mapping
     attribute(:Campuses)                              { campuses }
     # using server date time not utc, so it's local date time?
-    attribute(:ApplicationsAcceptedFrom)              { object.applications_open_from.to_date.strftime('%Y-%m-%dT%H:%M:%S') }
+    attribute(:ApplicationsAcceptedFrom)              { object.applications_open_from&.to_date&.strftime('%Y-%m-%dT%H:%M:%S') }
```

Comment:
Similar to the guard on `.utc`, I would avoid guarding against `.to_date` returning nil ... if it does, I think that's an error and we should find out about it.

Reply:
this one is not a error

`object.applications_open_from`

that derives from campus/site/site_status/location 

whereby the user can set the `actual application open date`

so the output i want is either the date or nil

@benilovj to confirm

Reply:
Similar to above, I think my comment was partly down to forgetting how `&.` works exactly.